cloudmarker/stores/emailstore.py

```diff
@@ -0,0 +1,68 @@
+"""email store plugin."""
+
+
+import logging
+from email.mime.multipart import MIMEMultipart
+from email.utils import COMMASPACE, formatdate
+from smtplib import SMTP, SMTPException
+
+# Define module-level logger.
+_log = logging.getLogger(__name__)
+
+
+class EmailStore():
```

Comment:
Since this class is not interiting some other class,  this class declaration can be made like : `class EmailAlert:` Without using `()`.

Reply:
Done